doc: prepare new release 3.7.0 (#4399)
tracim/tracim: changelog.md

# 3.7.0 / 2021-04-02

### New Features

> - You can create your own publications to share informations with other member without creating a specific content (like a blog page)

Reviewer: ```suggestion
- You can create your own publications to share information with other members without creating a specific content (like a blog page)
```

> - You can add reactions to contents using emojis like usable in social networking platform

Reviewer 2: ```suggestion
- You can add reactions to contents using emojis like in a social networking platform
```

Author: thanks

> - You can add reactions to contents using emojis like usable in social networking platform

Reviewer 3: ```suggestion
- You can add reactions to contents using emojis like you can find on social networking platforms
```

Author: thanks

> - You can add reactions to contents using emojis like usable in social networking platform

Reviewer: ```suggestion
- You can add reactions to contents using emojis the same way as in other social networking platforms or instant messaging tools
```

Author: thanks

> - The activity feed of space is now visible in dashboad and replace recent activity view

Reviewer 3: ```suggestion
- The activity feed of a space is now visible in dashboad and replace recent activity view
```

### Fixed Issues

- Frontend: [#4341](https://github.com/tracim/tracim/issues/4341)

### Breaking Changes

- Database: databases that were created using Tracim v1 need to be updated manually, [more information here](https://github.com/tracim/tracim/issues/4282#issuecomment-801944438) (issue [#4266](https://github.com/tracim/tracim/issues/4266))

### Other Changes

> - Reaction feature add big change to mysql/mariadb support: database migration needed for mysql/mariadb with [this tracimcli command line](https://github.com/tracim/tracim/blob/2a6aae281046e7a10bf4c5b24dfc60f81214c709/backend/doc/cli.md#migrate-mysqlmariadb-database-to-utf8mb4-added-for-tracim-37) (issue [#4272](https://github.com/tracim/tracim/issues/4272))

Reviewer: ```suggestion
- The Reaction feature requires a migration when using a MySQL or MariaDB database. This migration can be applied with [this tracimcli command line](https://github.com/tracim/tracim/blob/2a6aae281046e7a10bf4c5b24dfc60f81214c709/backend/doc/cli.md#migrate-mysqlmariadb-database-to-utf8mb4-added-for-tracim-37) (issue [#4272](https://github.com/tracim/tracim/issues/4272))
```

Author: I used the grignards wording.

> - Reaction feature add big change to mysql/mariadb support: database migration needed for mysql/mariadb with [this tracimcli command line](https://github.com/tracim/tracim/blob/2a6aae281046e7a10bf4c5b24dfc60f81214c709/backend/doc/cli.md#migrate-mysqlmariadb-database-to-utf8mb4-added-for-tracim-37) (issue [#4272](https://github.com/tracim/tracim/issues/4272))

Reviewer 3: ```suggestion
- This new version requires a special migration for mysql/mariadb (due to the reaction feature): [tracimcli command line](https://github.com/tracim/tracim/blob/2a6aae281046e7a10bf4c5b24dfc60f81214c709/backend/doc/cli.md#migrate-mysqlmariadb-database-to-utf8mb4-added-for-tracim-37) (issue [#4272](https://github.com/tracim/tracim/issues/4272)).
```

- API: this API`/api/workspaces/<workspace_id>/contents` now returns an object containing the contents in "items" instead of a direct list and also contains pagination information when the count/page_token parameters are used (issue [#4264](https://github.com/tracim/tracim/issues/4264))

> - Modification of default value used in development.ini: default value for `web.notifications.excluded`as changed to a reduced synonym for user and add filtering for new type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272))

Reviewer: ```suggestion
- Modification of a default value used in development.ini: default value for `web.notifications.excluded`as changed to a reduced synonym for user and add filtering for new type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272))
```

Author: done

> - Modification of default value used in development.ini: default value for `web.notifications.excluded`as changed to a reduced synonym for user and add filtering for new type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272))

Reviewer 3: ```suggestion
- Modification of a default value used in `development.ini`: default value for `web.notifications.excluded` has changed to a simpler syntax for user and adds filtering of the new type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272))
```

> - Modification of default value used in development.ini: default value for `web.notifications.excluded`as changed to a reduced synonym for user and add filtering for new type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272)).

Reviewer 3: ```suggestion
- Modification of a default value in `development.ini`: the default value for `web.notifications.excluded` has changed to a simple syntax for `user` and adds filtering for new the new notification type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272)).
```

>   - On existing Tracim installations, also if using the docker image: you can update your development.ini (use this file [development.ini.sample](backend/development.ini.sample) to compare).

Reviewer: ```suggestion
  - On existing Tracim installations, also if using the docker image: you may want to update your development.ini configuration file (use this file [development.ini.sample](backend/development.ini.sample) to compare).
```

>   - On existing Tracim installations, also if using the docker image: you may want to update your development.ini configuration file (use this file [development.ini.sample](backend/development.ini.sample) to compare).

Reviewer 3: ```suggestion
  - On existing Tracim installations, including the official docker image: you may want to update your `development.ini` configuration file (use this file [development.ini.sample](backend/development.ini.sample) to compare).
```
